Tests/MessagingInApp/Views/InAppMessageViewTest.swift

```diff
@@ -47,11 +43,11 @@ class InAppMessageViewTest: UnitTest {
 
         let view = InAppMessageView(coder: EmptyNSCoder())!
 
-        XCTAssertFalse(isDisplayingInAppMessage(view)) // Assert View is in dismissed state
+        XCTAssertFalse(isInlineViewVisible(view)) // Assert View is in dismissed state
```

Comment:
I decided to rename this test function to be more accurate for what the function checks. 

When I was reading `isDisplayingInAppMessage`, I read that name and thought that it was also checking a specific in-app message was being shown, which is not true. You must also call `getInAppMessage` to check that.